docs/user-guide/src/library.md

```diff
@@ -27,3 +29,32 @@ To get started, add the following to your `Cargo.toml` file:
 [dependencies]
 datafusion = "5.1.0"
 ```
+
+## Optimized Configuration
+
+For an optimized build several steps are required. First, use the following in your `Cargo.toml`:
+
+```toml
+[dependencies]
+datafusion = { git = "https://github.com/apache/arrow-datafusion.git", features = ["simd"]}
```

Comment:
```suggestion
datafusion = "5.0"
```

Reply:
we dont need to keep the `features = ["simd"]`?

Reply:
Ah you're right 👍 But probably we don't want people building from git, if not needed.